css-viewport/Overview.bs

```diff
@@ -260,6 +261,85 @@ as follows:
 
 Issue: Specify extend-to-zoom behavior by the viewport meta tag
 
+<h3 id="interactive-widgets-section">''interactive-widgets''</h3>
+
+Issue: Move the definition of ''visual viewport'' from CSSOM-View to this spec.
+
+The <dfn><code>interactive-widgets</code></dfn> property specifies the effect that interactive UI
+widgets have on the page's viewports. It defines whether widgets overlay a given viewport or whether
+the viewport is shrunken so that it remains fully visible while the widget is showing. Interactive
+UI widgets are transient user agent or operating system UI through which a user can provide input.
+
+<div class="note">The most common such UI widget is a virtual keyboard.</div>
+
+The following is a list of valid values for [=interactive-widgets=] and the associated
+viewport-resizing behavior:
+
+<dl>
+	<dt><dfn><code>overlays-content</code></dfn></dt>
+	<dd>
+		Interactive UI widgets MUST NOT [=resize=] the [=initial viewport=] nor
+		the <a spec="CSSOM-VIEW">visual viewport</a>. The user agent must perform the same steps
+		<!--TODO I can't seem to autolink to overlaysContent, help?-->
+		as when <a href="https://www.w3.org/TR/virtual-keyboard/#dom-virtualkeyboard-overlayscontent">
+		VirtualKeyboard.overlaysContent</a> is set to <code>true</code>.
+	</dd>
+	<dt><dfn><code>resize-layout</code></dfn></dt>
```

Comment:
So https://github.com/web-platform-tests/interop-2022-viewport/issues/25 is a discussion about whether the "layout viewport" should be called something else... Maybe we can give it a bit more thought and figure out whether we want to change that? If we do want to change that, it seems using here the layout term would be unfortunate...

Reply:
Hmm, how about "resize-initial" since that's what the spec text refers to

Reply:
`resize-initial` seems weird to me, as that reminds me of the initial in ICB, whereas this targets the LVP. Just commented on https://github.com/web-platform-tests/interop-2022-viewport/issues/25 and suggested “Document Viewport” to further specify it without introducing a new ambiguous term. Should suggestion make it in, maybe `resize-document` could be considered?

Reply:
Well, it does resize the ICB which I think is the most salient part of its behavior (it also happens to resize the fixed viewport).

That said, "Document Viewport" actually sgtm, will reply on that issue.

Reply:
There was a good counter point to "document viewport" on that issue but I also realized that this really does resize more than _just_ this viewport. In skobe's example page on https://github.com/web-platform-tests/interop-2022-viewport/issues/25, this mode will resize both LV and ICB. This really does resize the "initial viewport", though I agree it's not a great author-facing name.

How about we don't make it "viewport" specific and describe it in less precise but terms more understandable to authors? Some other ideas:

* `resizes-content` works well as a contrast to `overlays-content` I think
* `resizes-container`
* `resizes-window`

Reply:
No hard feelings about `resizes-content`, although technically your content does not get resized. I mean, your content will still have the same size when the OSK gets shown, unless you have them sized to specific viewport units.

Definitely not a fan of `resizes-container` given we have [Container Queries in CSS](https://drafts.csswg.org/css-contain-3/#container-queries). This would lead to confusion.

Not a huge fan of `resizes-window` as it does not actually resize the app’s window, right? I would think an app _(on mobile)_ to still run at the same size as on launch when the OSK gets shown. Is this the case? [By the looks of it](https://user-images.githubusercontent.com/213073/191926687-57300f61-9aa9-409d-a9e5-a096bc96c19e.png), this is also the behavior on Windows _(at least for Chrome)_.

Reply:
Good points about `container` and `window`.

> I mean, your content will still have the same size when the OSK gets shown, unless you have them sized to specific viewport units.

I'd say it has the _potential_ to resize content since it resizes the initial containing block.

@emilio WDYT?

Reply:
`resizes-content` works for me... It might be worth raising this in the next meeting to get wider feedback tho.